src/psyclone/tests/config_test.py

```diff
@@ -301,152 +323,139 @@ def test_dm():
 
 def test_api_not_in_list(tmpdir):
     ''' Check that we raise an error if the default API is not in
-    the list of supported APIs '''
+    the list of supported APIs.
+
+    '''
+    config_file = tmpdir.join("config")
     content = re.sub(r"^API = .*$",
                      "API = invalid",
                      _CONFIG_CONTENT,
                      flags=re.MULTILINE)
     config_file = tmpdir.join("config")
-    with config_file.open(mode="w") as new_cfg:
-        new_cfg.write(content)
-        new_cfg.close()
-        config = Config()
-        with pytest.raises(ConfigurationError) as err:
-            config.load(config_file=str(config_file))
 
-        assert ("The API (invalid) is not in the list of "
-                "supported APIs" in str(err.value))
+    with pytest.raises(ConfigurationError) as err:
+        config(config_file, content).load(config_file=str(config_file))
```

Comment:
This and all other uses of config can be simplified to
```
config(config_file, content)
```
after the previous suggested change has been done.

Reply:
Done.